src/types/Message.ts

```diff
@@ -0,0 +1,184 @@
+import type { ITerms } from './Terms'
+import type { ICredential, ICredentialPresentation, IAttestation, CTypeHash, DidResourceUri, DidUri, IDelegationNode,PartialClaim } from '@kiltprotocol/types'
+import type { IQuoteAgreement } from './Quote'
+
+
+export interface IDelegationData {
```

Comment:
document this

Reply:
Is this actually used anywhere? Do we need this?

Reply:
I removed the type.